docs/source/markdown/podman-machine.1.md

```diff
@@ -11,15 +11,16 @@ podman\-machine - Manage Podman's virtual machine
 
 ## SUBCOMMANDS
 
-| Command | Man Page                                                | Description                       |
-| ------- | ------------------------------------------------------- | --------------------------------- |
-| init    | [podman-machine-init(1)](podman-machine-init.1.md)      | Initialize a new virtual machine  |
-| list    | [podman-machine-list(1)](podman-machine-list.1.md)      | List virtual machines             |
-| rm      | [podman-machine-rm(1)](podman-machine-rm.1.md)          | Remove a virtual machine          |
-| set     | [podman-machine-set(1)](podman-machine-set.1.md)        | Sets a virtual machine setting    |
-| ssh     | [podman-machine-ssh(1)](podman-machine-ssh.1.md)        | SSH into a virtual machine        |
-| start   | [podman-machine-start(1)](podman-machine-start.1.md)    | Start a virtual machine           |
-| stop    | [podman-machine-stop(1)](podman-machine-stop.1.md)      | Stop a virtual machine            |
+| Command | Man Page                                             | Description                       |
+|---------|------------------------------------------------------|-----------------------------------|
+| init    | [podman-machine-init(1)](podman-machine-init.1.md)   | Initialize a new virtual machine  |
+| inspect | [podman-machine-inspect(1)](podman-machine-inspect.1.md)  | Inspect one or more virtual machines |
+| list    | [podman-machine-list(1)](podman-machine-list.1.md)   | List virtual machines             |
+| rm      | [podman-machine-rm(1)](podman-machine-rm.1.md)       | Remove a virtual machine          |
+| set     | [podman-machine-set(1)](podman-machine-set.1.md)     | Sets a virtual machine setting    |
+| ssh     | [podman-machine-ssh(1)](podman-machine-ssh.1.md)     | SSH into a virtual machine        |
+| start   | [podman-machine-start(1)](podman-machine-start.1.md) | Start a virtual machine           |
+| stop    | [podman-machine-stop(1)](podman-machine-stop.1.md)   | Stop a virtual machine            |
```

Comment:
Should we be uber specific and say "Podman virtual machine"?  Will someone be silly enough to complain because the command doesn't work on a VMware virtual machine?